admin_manual/configuration/files/federated_cloud_sharing_configuration.rst

```diff
@@ -7,6 +7,8 @@ now called Federation sharing. When you enable the Federation app you can
 easily and securely link file shares between ownCloud servers, in effect 
 creating a cloud of ownClouds.
 
+For security reasons federated sharing **strictly requires HTTPS** (SSL/TLS).
```

Comment:
My only suggestion is move "(SSL/TLS)" inside the bolded text preceding it. Otherwise, looks good.

Reply:
Ok